PSQL_README.md

```diff
@@ -0,0 +1,53 @@
+# How to Install PSQL
+## linux
```

Comment:
this works only on Debian and its children releases. `linux` is kind of wrong usage.